shell.py

```diff
@@ -422,6 +422,12 @@ def wait_install(type, regions=use_regions(), tag='dev'):
 # ======================================================================================
 
 
+def upgrade_binary(regions, tag='dev', delay=10):
```

Comment:
It is just a note. Since AlephBFT needs at least 5 seconds to start, maybe we would like to add some more time here. But also I'm thinking that 10 secends are just ok as lowerbound

Reply:
I've added manual confirmation as an option.